BuildForPublication.cmd

```diff
@@ -3,12 +3,21 @@ goto :main
 
 :main
 setlocal
+  set errorlevel=
+  set BuildConfiguration=Release
+  set VersionSuffix=build0047
+  set PackageVersion=1.0.0-alpha-%VersionSuffix%
 
-set PackageVersion=1.0.0-alpha-build0043
-set VersionSuffix=build0043
+  echo/==================
+  echo/ Building version %PackageVersion% NuGet packages.
+  echo/==================
 
-echo Building version %PackageVersion% NuGet packages.
+  set LocalDotNet_ToolsDir=%~dp0tools
+  if exist "%LocalDotNet_ToolsDir%" rmdir /s /q "%LocalDotNet_ToolsDir%"
```

Comment:
I think that doing git clean -fxd is probably the right thing, but I also think that we should prompt the user with a warning before doing it.  It would be easy for someone to run BuildForPublication without knowing that it's going to blow away their untracked changes.